qiskit-superstaq/example-requirements.txt

```diff
@@ -1,4 +1,4 @@
 matplotlib>=3.7.0
 notebook>=6.5.5
 pylatexenc>=2.0
-qiskit-ibm-provider>=0.11.0
+qiskit-ibm-runtime>=0.27.0
```

Comment:
I think we may need to update the other [DD notebook](https://github.com/Infleqtion/client-superstaq/blob/main/docs/source/optimizations/ibm/ibmq_dd.ipynb) as well before we drop `qiskit-ibm-provider`. 

(An aside, but I feel like this notebook should also show up on the readthedocs)

Reply:
> I think we may need to update the other [DD notebook](https://github.com/Infleqtion/client-superstaq/blob/main/docs/source/optimizations/ibm/ibmq_dd.ipynb) `try..except` install

done. (though that notebook also really needs to be updated - as is it won't run even with the correct installs)

> (An aside, but I feel like this notebook should also show up on the readthedocs)

good idea, done.

Reply:
>  as is it won't run even with the correct installs

That's a valid point lol, I guess somewhat for the sake of principle and working towards the eventual update. Will make an issue as a reminder